homeassistant/components/met/weather.py

```diff
@@ -286,4 +286,5 @@ def device_info(self) -> DeviceInfo:
             identifiers={(DOMAIN,)},  # type: ignore[arg-type]
             manufacturer="Met.no",
             model="Forecast",
+            configuration_url="https://www.met.no",
```

Comment:
Should we point this to the English version of the website instead?

Reply:
I did wonder, hence the comment. I'll point to the English version of the site instead if you prefer